ouroboros-consensus-test/test-consensus/Test/Consensus/HardFork/Combinator/A.hs

```diff
@@ -199,8 +198,24 @@ data PartialLedgerConfigA = LCfgA {
     }
   deriving NoThunks via OnlyCheckWhnfNamed "LCfgA" PartialLedgerConfigA
 
-type instance LedgerCfg (LedgerState BlockA) =
-    (EpochInfo Identity, PartialLedgerConfigA)
```

Comment:
Was this `EpochInfo Identity` just dead-code?

Reply:
Yes, as far as I can see.

Reply:
That's a good observation. But I think we should remove the dead-code.

If we want to emphasize the `EpochInfo` thing here, then we should figure out a way to make it not dead -- ie to actually need it.

Reply:
I'm leaning toward keeping the current change (i.e. removing the `EpochInfo` from the partial ledger config). In the case of `ByronBlock` we also don't add an `EpochInfo` in it's partial ledger config, hence there is precedent.

Reply:
> But I think we should remove the dead-code.

I'm reading this as "we should remove the `EpochInfo` from `LedgerCfg (LedgerState BlockA)`". This is in fact what I've done. Is there something else you want changed here?

Reply:
Yeah; maybe the code wasn't as fixed-up when I originally wrote that? Or maybe I was confused.

It looks good now. Resolving. Thanks.